zypp/RepoManager.cc

```diff
@@ -1238,6 +1238,9 @@ namespace zypp
   void RepoManager::Impl::buildCache( const RepoInfo & info, CacheBuildPolicy policy, const ProgressData::ReceiverFnc & progressrcv )
   {
     assert_alias(info);
+    cleanCache(info);
+    refreshMetadata(info, RefreshForced);
+
```

Comment:
Stupid mistake. Leftover from debugging that I failed to remove from this commit. I'll fix this along with the other comments and update the PR soon.